docs/websocket/speech_recording.md

```diff
@@ -106,3 +106,63 @@ The RPC can return various errors:
 
 * `nl.itslanguage.no_such_organisation` If the organisation doesn't exist.
 * `nl.itslanguage.no_such_challenge` If the challenge doesn't exist.
+
+
+## Feedback
+
+It's possible to get feedback while recording. After every sentence feedback is
+provided indicating whether or not the sentence was read well.
+
+### URI
+
+```
+nl.itslanguage.recording.feedback
+```
+
+### Parameters
+
+Name         | Type     | Description
+-------------|----------|------------
+recording_id | `string` | **Required** The unique id of the speech recording.
+rpc          | `string` | **Required** The URI of a [registered audio rpc](wamp.md#register-audio-procedure).
+
+### Response
+
+The rpc returns progressive results for realtime feadback. After every sentence
+the following json is sent as a progressive result:
+
+```json
+{
+  "sentence": 0,
+  "feedback": 1
+}
+```
+
+Name     | Type  | Description
+---------|-------|------------
+sentence | `int` | The index of the sentence, starting at `0`.
+feedback | `int` | Feedback score, `0` means bad and `1` means good.
```

Comment:
perhaps use a bool: recognitionAccepted: true/false

Reply:
CLD returns a float as feedback (same as here). I figured that it might in the future indicate how well something was pronounced, like `0.6` for an okay pronunciation. If this is the case a bool doesn't suffice.
Any thoughts on this?

Reply:
Well, that bool makes more sense in terms of incremental development.

Feedback will be more than a float because it will be more than just bad/good and an percentage. At least, that is the required situation.